src/Config.js

```diff
@@ -16,10 +16,14 @@ const consts = {
   main: {
     tokenUriStart: "https://main.net955305.contentfabric.io/s/main/q/",
     tokenUriEnd: "/meta/public/nft",
+    kmsAddress: "0xd962aff088ca845de83ce0db0c91b9a0b93d294f",
+    spaceAddress: "0x6612d94a31fab146b4c7ace60ddf3a1e5e40e500",
```

Comment:
space address can be derived from /config (can be removed in a separate PR)

Reply:
ok will do next PR